src/utils/tree-transform.js

```diff
@@ -72,6 +74,35 @@ const getDataMatrix = async (layout, model) => {
   return dataMatrix;
 };
 
+function getAttributIndex(attrsInfo, id) {
+  if (attrsInfo && attrsInfo.length) {
+    let index = false;
+    attrsInfo.forEach((attr, i) => {
+      if (attr.id === id) {
+        index = i;
+      }
+    });
+    return index;
+  }
+  return false;
+}
+
+function getAttribute(attrs, index) {
+  if (index !== false) {
+    return attrs.qValues[index].qText;
+  }
+  return false;
+}
+
+function transformColor(color) {
```

Comment:
We have a bunch of stuff in the button repo handling color, if it's needed :)

Reply:
yea, basically I just solved the ARGB case that engine spits out, but there are likely more things. And if we already solved them, we could pick them from there. 
This is probably something that should end up in nebula later.

Reply:
ah true that would be really nice